app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less

```diff
@@ -84,6 +84,12 @@
             content: '';
         }
     }
+
+    &.active {
+        &:before {
+            transform: rotate(180deg);
```

Comment:
Would also recommend to use content attribute instead and change icon variable

Reply:
Ok @Karlasa , 

Let me check the solution by using the content attribute.